### PR TITLE
Decompile starter choose

### PR DESCRIPTION
**Pull request by DizzyEggg**

### CODE REVIEW

File: data/starter_choose.s [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+gUnknown_085B1DE4:: @ 85B1DE4
+	.incbin "baserom.gba", 0x5b1de4, 0x8
+
+sPokeballCoords:: @ 85B1DEC
+	.incbin "baserom.gba", 0x5b1dec, 0x6
+
+gStarterChoose_LabelCoords:: @ 85B1DF2
+	.incbin "baserom.gba", 0x5b1df2, 0x6
+
+sStarterMons:: @ 85B1DF8
```
Comment (Diegoisawesome) [inline data/starter_choose.s]: Mons->Mon

File: data/starter_choose.s [hunk 2]
```diff
@@ -0,0 +1,67 @@
+@ the fourth big chunk of data
```
Comment (Diegoisawesome) [inline data/starter_choose.s]: Why isn't all of this data properly extracted into the C file?

File: src/starter_choose.c [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+#include "international_string_util.h"
+#include "trig.h"
+
+#define STARTER_MON_COUNT   3
+
+// Position of the sprite of the selected starter Pokemon
+#define STARTER_PKMN_POS_X 120
+#define STARTER_PKMN_POS_Y 64
+
+extern u16 sStarterChooseWindowId;
```
Comment (Diegoisawesome) [inline src/starter_choose.c]: Shouldn't be externed, this file should be included in sym_bss.

File: data/starter_choose.s (hunk 2 above)
Comment (DizzyEggg) [reply data/starter_choose.s]: Because I'm too lazy to dump graphics. :((

File: sym_bss.txt [hunk 4]
```diff
@@ -55,8 +55,8 @@ gUnknown_030011F6: @ 30011F6
 gUnknown_030011F7: @ 30011F7
 	.space 0x1
 
-sStarterChooseWindowId: @ 30011F8
-	.space 0x4
+    .include "src/starter_choose.o"
+    .space 0x2
```
Comment (Diegoisawesome) [inline sym_bss.txt]: Don't use .space to align, use .align

**Pull request merged**